TestProject.test_conf_file_not_found fails on master

## Details

With `09f8282be35c20dcca42c85d3f845bba8b784cfe`
```
tox -e py27
```
fails

## Expected Result

Tests green

## Actual Result


```
TestProject.test_conf_file_not_found 

self = <readthedocs.rtd_tests.tests.test_project.TestProject testMethod=test_conf_file_not_found>

    def test_conf_file_not_found(self):
        with self.assertRaisesMessage(
                ProjectConfigurationError,
                ProjectConfigurationError.NOT_FOUND) as cm:
>           self.pip.conf_file()

rtd_tests/tests/test_project.py:146: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/lib/python2.7/contextlib.py:24: in __exit__
    self.gen.next()
../.tox/py27/local/lib/python2.7/site-packages/django/test/testcases.py:616: in _assert_raises_message_cm
    yield cm
E   AssertionError: ProjectConfigurationError not raised
```

--- a/readthedocs/rtd_tests/tests/test_project.py
+++ b/readthedocs/rtd_tests/tests/test_project.py
@@ -139,7 +139,11 @@ class TestProject(TestCase):
             '/home/docs/rtfd/code/readthedocs.org/user_builds/pip/checkouts/latest/docs/conf.py',
         )
 
-    def test_conf_file_not_found(self):
+    @patch('readthedocs.projects.models.Project.find')
+    @patch('readthedocs.projects.models.Project.full_find')
+    def test_conf_file_not_found(self, find_method, full_find_method):
+        find_method.return_value = []
+        full_find_method.return_value = []
         with self.assertRaisesMessage(
                 ProjectConfigurationError,
                 ProjectConfigurationError.NOT_FOUND) as cm:

